1907-02-18.xml

```diff
@@ -0,0 +1,2349 @@
+
+                        <p>THE IMPERIAL FIRE OFFICE united with THE ALLIANCE ASSURANCE, Co.,
+                            Ltd.</p>
+
+                        <p>1, Old Broad Street, LONDON— Policies issued at SUEZ by G. BEYTS &amp;
+                            Co., Agents.</p>
+
+                    </div>
+                    <div type="advert" xml:id="deg-ad-tce01">
+
+                        <head>Telephone Company of Egypt, Limited.</head>
+
+                        <p>CAIRO-ALEXANDRIA TELEPHONE.--Rates as follows P.T. 5 for each 3 minutes,
+                            or fraction of 3 minutes; P.T. 10 for over 3 up to 8 minutes
+                            communication.</p>
+
+                        <p>PUBLIC CALL-OFFICES : Cairo, Central Office, Opera Square, and New Bar;
+                            Helouan, Central Office, Maison Purvis ; Alexandria, St Mark’s
+                            Buildings, Egyptian Bar, I. Castelli &amp; Co.; Ramleh, Central Office.
+                            San Stefano Casino <measure>30.4.907</measure></p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-tcs01">
+
+                        <head>Thos. Cook &amp; Son, (EGYPT, LTD)</head>
+
+                        <p>(EGYPT), LIMITED, HEAD OFFICE—LUDGATE CIRCUS—LONDON.</p>
+
+                        <p>CHIEF EGYPTIAN OFFICE — CAIRO, near SHEPHEARD'S HOTEL.</p>
+
+                        <p>Alexandria, Port-Said, Suez, Luxor, Assuan, Haifa, &amp; Khartum.</p>
+
+                        <p>GENERAL RAILWAY AND STEAMSHIP AGENTS. BANKERS.</p>
+
+                        <p>BAGGAGE AND FORWARDING AGENTS.</p>
+
+                        <p>Officially appointed &amp; Sole Agents in Cairo to the P.&amp;O. S.N.
+                            Co.</p>
+
+                        <p> NILE STEAMER SERVICES</p>
+                        <!-- Check formating because not a template and continue rest of ad -->
+                        <p> TOURIST SERVICE. The large and splendidly appointed S.S. Ramses III.,
+                            will leave Cairo on Tuesday, February 19th, for Luxor, Assuan and
+                            Philae. </p>
+                        <p> EXPRESS SERVICE. Steamers leave Cairo every Monday and Friday for Luxor,
+                            Assuan, and Philae. 19 days on the Nile for £22. </p>
+                        <p> WEEKLY SERVICE TO HALFA, KHARTOUM AND THE SOUDAN. </p>
+                        <p> Special combined rail and teamer Nile tours at greatly reduced prices. </p>
+                        <p> Special steamers and Dehabeahs for Private Parties. </p>
+                        <p> Special arrangements for tours in Palestine, Syria, and the Desert
+                            Lowest Rates. </p>
+                        <p> BEST OAMP EQUIPTMENT IN THE COUNTRY. </p>
+                    </div>
+                </div>
```

Comment:
The rest of your page 1 ads should go here, rather than in the page 3 division below. Don't confuse page divs with columns.